handler.cpp

```diff
@@ -273,4 +273,156 @@ RequestHandler::Status StatusHandler::HandleRequest(const Request& request, Resp
     return RequestHandler::PASS;
 }
 
+//////////////////////////////////////////////////////////////////////////////////////
+///////////////////////////////// Proxy Handler /////////////////////////////////////
+//////////////////////////////////////////////////////////////////////////////////////
+
+RequestHandler::Status ProxyHandler::Init(const std::string& uri_prefix, const NginxConfig& config) {
+
+    for (unsigned int i = 0; i < config.statements_.size(); i++){
```

Comment:
this is fine and will work, but consider using iterators in the future to iterate over a vector. I believe they are more efficient and likely to be correct independent of stl container.

i.e. for (some_iterator = some_vector.begin(); some_iterator != some_vector.end(); some_iterator++)